web/src/activity_ui.js

```diff
@@ -93,13 +93,14 @@ export function build_user_sidebar() {
         return undefined;
     }
 
-    const filter_text = get_filter_text();
+    const filter_text = user_filter ? user_filter.text() : "";
```

Comment:
Why is this change correct -- is it to avoid the `get_filter_text` blueslip warning? I'm wondering if there's a deeper issue with `activity_ui.initialize` being called too late that you're masking with this change.

Reply:
This is the trace where this comes up:

```js 
    build_user_sidebar activity_ui.js:97
    handle_post_view_change narrow.js:1034
    activate narrow.js:533
    do_hashchange_normal hashchange.js:159
    hashchanged hashchange.js:411
    initialize hashchange.js:424
    initialize_everything ui_init.js:669
    js ui_init.js:740
```

So it's from hashchange's initialize function. I'm going to add another commit after this main one to move the `hashchange` initialization after `activity_ui`, which doesn't cause tests to fail and hopefully won't cause other issues.